lib/spoom/sorbet/translate/sorbet_sigs_to_rbs_comments.rb

```diff
@@ -336,6 +334,13 @@ def delete_extend_t_generics
 
           @extend_t_generics.clear
         end
+
+        #: -> Array[[Prism::CallNode, RBI::Sig]]
+        def collect_last_sigs
```

Comment:
Maybe a comment here is useful that mentions resetting, it's a bit strange looking.